packages/asset-service/src/service/descriptions.json

```diff
@@ -1,6 +1,7 @@
 {
   "eip155:1/erc20:0xed4064f376cb8d68f770fb1ff088a3d0f3ff5c4d": "This token represents a liquidity pool on Curve, a leading decentralized exchange. Holders earn fees from users trading in the pool, and can also deposit the LP token on ShapeShift to earn yield. This liquidity pool contains CRV and ETH. To acquire Curve CRV-ETH, deposit CRV and/or ETH on https://curve.fi/crveth/deposit. Please be aware that as liquidity pools are composed of differently-priced assets, they are subject to impermanent loss.",
-  "eip155:1/erc20:0x5a6a4d54456819380173272a5e8e9b9904bdf41b": "This token represents a liquidity pool on Curve, a leading decentralized exchange. Holders earn fees from users trading in the pool, and can also deposit the LP token on ShapeShift to earn yield. This liquidity pool contains MIM. To acquire Curve MIM, deposit MIM on https://curve.fi/crvmim/deposit. As a single-sided staking pool, it is not subject to impermanent loss."
+  "eip155:1/erc20:0x5a6a4d54456819380173272a5e8e9b9904bdf41b": "This token represents a liquidity pool on Curve, a leading decentralized exchange. Holders earn fees from users trading in the pool, and can also deposit the LP token on ShapeShift to earn yield. This liquidity pool contains MIM and 3crv. 3CRV is another curve LP token that consists of DAI, USDT, and USDC.  To acquire crvMIM, deposit MIM, DAI, USDT, and/or USDC  on [Curve](https://curve.fi/mim/deposit).Please be aware that 
+as liquidity pools are composed of differently-priced assets, they are subject to impermanent loss but because all of the assets in the pool are pegged to $1, Impermanent Loss is unlikely and would only result if one of the assets in the pool lost its peg."

```

Comment:
```suggestion
  "eip155:1/erc20:0x5a6a4d54456819380173272a5e8e9b9904bdf41b": "This token represents a liquidity pool on Curve, a leading decentralized exchange. Holders earn fees from users trading in the pool, and can also deposit the LP token on ShapeShift to earn yield. This liquidity pool contains MIM and 3CRV. 3CRV is another curve LP token that consists of DAI, USDT, and USDC. To acquire crvMIM, deposit MIM, DAI, USDT, and/or USDC on [Curve](https://curve.fi/mim/deposit). Please be aware that 
as liquidity pools are composed of differently-priced assets, they are subject to impermanent loss but because all of the assets in the pool are pegged to $1, impermanent loss is unlikely and would only result if one of the assets in the pool lost its peg."
```